app/template_statistics/rest.py

```diff
@@ -29,17 +29,18 @@
 
 @template_statistics.route('')
 def get_template_statistics_for_service_by_day(service_id):
+    whole_days = request.args.get('whole_days', request.args.get('limit_days', ''))
```

Comment:
Is the plan to drop this fallback once admin is released or are there other things using it?

Reply:
nah - will drop this fallback. This endpoint is only used by the service dashboard page